src/app/components/MaskedImage/index.tsx

```diff
@@ -14,11 +14,13 @@ const MaskedImage = ({
   imageUrlTemplate,
   imageWidth,
   altText = '',
+  showPlaceholder = true,
```

Comment:
I wonder if this should be defaulted to `true` or not. The default for the Image component is `true` so an undefined value will become `true`... 🤔 https://github.com/bbc/simorgh/blob/802f5f1d48a82bb8f575d34d0eafa9d7a68d4671/src/app/components/Image/index.tsx#L53

Reply:
I think its ok to have it default to `true` here since it has been `true` anyway as you say.